src/utils.js

```diff
@@ -34,3 +34,23 @@ export function useInterval({ callback, delay, runOnStart }) {
     }
   }, [delay, runOnStart]);
 }
+
+export function isFeatureEnabled(feature) {
+  const experimentalFeatures = localStorage.getItem('dopplerExpermiental')
+    ? JSON.parse(localStorage.getItem('dopplerExpermiental'))
+    : null;
+  if (experimentalFeatures) {
+    return !!experimentalFeatures.features.find((item) => item === feature);
```

Comment:
I cannot imagine the structure of `experimentalFeatures` object, is it something like:

```json
{
  features: [ 'feature1', 'feature2'],
  data: {
    'feature1': {},
    'arbitraryName: 5
  }
}
```

What do you think about:

```json
{
  feature1: true,
  feature2: {
    customValue1: "Hola"
  },
  feature3: false,
  feature4: null,
  feature5: 0
}
```

(features4, features45, and features6 are the same than disabled)

Reply:
The structure is in the tests. I did it this way to make a simple initial approach. But maybe it is best to change it with params by feature.

Reply:
Do you not like a simple object where the property names are the features and the property values the feature data?